inform/InTest/IFInTest.m

```diff
@@ -98,19 +98,23 @@ - (BOOL) isRunning {
 	return theTask!=nil?[theTask isRunning]:NO;
 }
 
--(void) executeInTestWithArgs:(NSArray*) args {
+-(void) executeInTestForExtension: extensionPathName withArgs:(NSArray*) args {
```

Comment:
I'd recommend that you add a type declaration for `extensionPathName` so the compiler will warn if you pass an object of the wrong type.